api/bean/AppView.go

```diff
@@ -123,6 +123,9 @@ type AppDetailContainer struct {
 	LinkOuts                  []LinkOuts             `json:"linkOuts,omitempty"`
 	ResourceTree              map[string]interface{} `json:"resourceTree,omitempty"`
 }
+type Notes struct {
+	Notes string `json:"gitOpsNotes,omitempty"`
```

Comment:
why did you create new struct for only notes, wasn't there any notes field in the response object?